### PR TITLE
Release Polish

### PR DESCRIPTION
**Pull request by rullzer**
For: https://github.com/nextcloud/server/issues/735

* Save next to title
* Slide extra options in/out

CC: @jancborchardt @LukasReschke @schiessle @icewind1991 @blizzz @MorrisJobke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Backport to stable10?

File: templates/admin.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 /** @var array $_ */
 ?>
 <form id="user-saml" class="section" action="#" method="post">
-	<h2><?php p($l->t('SAML')); ?></h2>
+	<h2 class="inlineblock"><?php p($l->t('SAML')); ?></h2>
```
Comment (blizzz) [inline templates/admin.php]: what is the inlineblock for? Other templates don't have it.

Comment (MorrisJobke) [inline templates/admin.php]: This allows the save notification to appear on the same level right beside the heading

**Comment by LukasReschke** [comment]
> Backport to stable10?

Already done. https://github.com/nextcloud/user_saml/commits/stable10

**Comment by MorrisJobke** [comment]
> Already done. https://github.com/nextcloud/user_saml/commits/stable10

No PR 🙈

**Pull request merged**